tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_DuDt.cpp

```diff
@@ -0,0 +1,138 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Equations.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+template <typename Tensor>
+Tensor create_tensor_with_random_values(
+    const size_t n_pts, const gsl::not_null<std::mt19937*> gen) {
+  Tensor tensor(n_pts);
+  for (auto& data : tensor) {
+    for (size_t s = 0; s < data.size(); ++s) {
+      data[s] = std::uniform_real_distribution<>(-10, 10)(*gen);
+    }
+  }
+  return tensor;
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.DuDt",
+                  "[Unit][GeneralizedHarmonic]") {
+  const size_t dim = 3;
+  const size_t n_pts = 1;
+
+  // This deliberately seeds the RNG with a constant value. This test compares
+  // the output of the GeneraralizedHarmonic RHS as compared to SpEC for the
+  // specific values generated by this seed.
```

Comment:
"This test compares the output of the GeneralizedHarmonic RHS to the output computed using SpEC for the specific input tensors generated by this seed."